.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           java-version: ${{ matrix.java-version }}
       - name: Download all Trino dependencies to make sure cache is complete
         run: |
-          $RETRY $MAVEN dependency:go-offline
+          $RETRY ./mvnw dependency:go-offline
```

Comment:
Does this defeat the purpose of having MAVEN variable?

Reply:
I'm using `--offline` only as an experiment to see if we actually downloaded all dependencies.